tests/release/pipelines/fbc_release.go

```diff
@@ -43,6 +43,10 @@ var (
 	snapshot *appservice.Snapshot
 	releaseCR *releaseapi.Release
 	pipelineRun *pipeline.PipelineRun
```

Comment:
Would it make sense to rename also this one (if still needed)?

Reply:
Renamed it to "managedPipelineRun", thx